packages/@react-spectrum/tabs/src/Tabs.tsx

```diff
@@ -280,9 +279,10 @@ export function TabList<T>(props: SpectrumTabListProps<T>) {
           'spectrum-Tabs--quiet': isQuiet,
           ['spectrum-Tabs--compact']: density === 'compact'
         },
-        styleProps.className
+        orientation === 'vertical' && styleProps.className
```

Comment:
Ah good catch with this style props change, forgot the horizontal orientation has a wrapper div.